<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>logindlg</class>
 <widget class="QDialog" name="logindlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_id">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>67</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_id">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_pw">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Admin_login">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>管理员登陆</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_pw">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>260</y>
     <width>41</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="User_login">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>会员登陆</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>logindlg</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>381</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>AdminLogin()</slot>
  <slot>UserLogin()</slot>
 </slots>
</ui>
